crates/core/transaction/src/action_list.rs

```diff
@@ -230,46 +228,3 @@ impl ActionList {
             .retain(|_, output| output.value.amount > Amount::zero());
     }
 }
-
-/// Prioritize notes to spend to release value of a specific transaction.
-///
-/// Various logic is possible for note selection. Currently, this method
-/// prioritizes notes sent to a one-time address, then notes with the largest
-/// value:
-///
-/// - Prioritizing notes sent to one-time addresses optimizes for a future in
-/// which we implement DAGSync keyed by fuzzy message detection (which will not
-/// be able to detect notes sent to one-time addresses). Spending these notes
-/// immediately converts them into change notes, sent to the default address for
-/// the users' account, which are detectable.
```

Comment:
i think i missed the discussion, but for posterity's sake: why are we no longer prioritize one-time addresses?

Reply:
For instance, selecting the largest notes up-front doesn't necessarily reduce gas usage overall -- the smaller notes will remain unspent and will be spent later and the gas will be paid then. Spending all the small notes up front sweeps them into change, which might be desirable anyways.

Reply:
that's true, good point

Reply:
marking as resolved via 3294222